rsearch/Godeps/Godeps.json

```diff
@@ -0,0 +1,23 @@
+{
+	"ImportPath": "github.com/romana/contrib/rsearch",
+	"GoVersion": "go1.5",
+	"GodepVersion": "v58",
+	"Deps": [
```

Comment:
Other things in contrib have actually committed the `_workspace` that godeps produces.
Might be necessary for this also.